elastic/logs/challenges/logging-querying-patterned-text.json

```diff
@@ -0,0 +1,132 @@
+{% import "rally.helpers" as rally %}
+{
+  "name": "logging-querying-patterned-text",
```

Comment:
I think we can include these queries into the `logging-querying` challenge (behind the `p_include_esql_queries` flag)? Then we don't need another nightly config and can just update the existing dashboard?